Rocket.Chat/Controllers/Chat/ChatDataController.swift

```diff
@@ -87,12 +87,11 @@ final class ChatDataController {
                     // Check if already contains some separator with this data
                     // swiftlint:disable for_where
```

Comment:
Can you remove this `swiftlint` comment?

Reply:
Done. Thank you. Didn't see that 😄